packages/squiggle-lang/src/fr/calculator.ts

```diff
@@ -0,0 +1,60 @@
+import { makeDefinition } from "../library/registry/fnDefinition.js";
+import {
+  frDict,
+  frLambda,
+  frArray,
+  frString,
+  frOptional,
+} from "../library/registry/frTypes.js";
+import { FnFactory } from "../library/registry/helpers.js";
+import { vCalculator } from "../value/index.js";
+
+const maker = new FnFactory({
+  nameSpace: "Calculator",
+  requiresNamespace: true,
+});
+
+export const library = [
+  maker.make({
+    name: "make",
+    output: "Calculator",
+    examples: [],
+    definitions: [
+      makeDefinition(
+        [
+          frDict(
+            ["fn", frLambda],
+            ["description", frOptional(frString)],
+            [
+              "fields",
+              frArray(
+                frDict(
+                  ["name", frString],
+                  ["default", frOptional(frString)],
```

Comment:
I could imagine changing this later to also allow for frNumber, but that would be some more work, and might have complications.